routes/v1.js

```diff
@@ -1631,28 +1631,21 @@ api.declare({
   route:      '/pending/:provisionerId/:workerType',
   name:       'pendingTasks',
   stability:  base.API.stability.stable,
-  scopes:     [['queue:pending-tasks:<provisionerId>/<workerType>']],
-  deferAuth:  true,
   output:     'pending-tasks-response.json#',
   title:      "Get Number of Pending Tasks",
   description: [
-    "Documented later...",
-    "This probably the end-point that will remain after rewriting to azure",
-    "queue storage...",
+    "Get an approximate number of pending tasks for the given `provisionerId`",
+    "and `workerType`.",
     "",
+    "The underlying Azure Storage Queues only promises to give us an estimate.",
```

Comment:
Hrm, do we need to provide this level of details in our user facing documentation?  Seems like users don't care what Azure storage queue is or why this is just an estimate.